programs/pkey/rsa_sign_pss.c

```diff
@@ -115,7 +115,13 @@ int main( int argc, char *argv[] )
         goto exit;
     }
 
-    mbedtls_rsa_set_padding( mbedtls_pk_rsa( pk ), MBEDTLS_RSA_PKCS_V21, MBEDTLS_MD_SHA256 );
+    if( ( ret = mbedtls_rsa_set_padding( mbedtls_pk_rsa( pk ),
+                                         MBEDTLS_RSA_PKCS_V21,
+                                         MBEDTLS_MD_SHA256 ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  ! Invalid padding\n" );
```

Comment:
Minor: we know that our code is correct so this can't fail. “Padding not supported” would be a better message, although at this time `mbedtls_rsa_set_padding` doesn't actually fail in this case.